### DISCUSSION TITLE
When a comment is "selected as the answer" by non-OP, should the OP receive an email about that?

### DISCUSSION THREAD

**Discussion by gagliardetto** [community/community#45] [Discussions] [answered]
Example:

- I opened https://github.com/github/securitylab/discussions/197
- I got a few comments (:heavy_check_mark:  got an email for each).
- A specific comment was "selected as the answer" by `@xcorail`; but that did not trigger an email towards me (:x:).

Was an email sent and I did simply not receive it? Or that is not a feature, yet? Thanks!

**Comment by becca** [comment, marked as answer]
👋 @gagliardetto thank you for opening this feedback discussion! it's also great to see your contributions in `github/securitylab` 🎉 

We do not send out any additional notifications right now for the answering/unanswering event. What were you hoping would happen when your answer was selected by someone from the securitylab team?

**Reply by gagliardetto** [reply]
Hi @becca Thank you for your reply!

I'd like to highlight the fact that the **selected answer** was not written by me (I was the one who **asked the question**).

I was hoping that I would be notified (being the OP of the question) when an answer was chosen as **"the answer"**, completing the cycle I set in motion (after all, I'm the one looking for an answer).

Having to check on the discussion page for whether an answer was selected or not yet is somewhat unexpected in the context of the GitHub experience.

**Reply by becca** [reply]
You should be receiving a notification, as a general rule, for any reply to a comment you leave (on Issues and Discussions). We don't send out any _additional_ notifications saying that a comment was a selected answer.

**Reply by gagliardetto** [reply]
Alright. Thanks for the definitive answer!

I don't use it often enough to be a problem to me, but it looked like a nice feature to have for users that do.
